lib/slack-ruby-bot/app.rb

```diff
@@ -3,26 +3,36 @@
 module SlackRubyBot
   class App < Server
     def initialize(options = {})
-      SlackRubyBot.configure do |config|
-        config.token = ENV['SLACK_API_TOKEN'] || raise("Missing ENV['SLACK_API_TOKEN'].")
-        config.aliases = ENV['SLACK_RUBY_BOT_ALIASES'].split(' ') if ENV['SLACK_RUBY_BOT_ALIASES']
-      end
-      Slack.configure do |config|
-        config.token = SlackRubyBot.config.token
-      end
-      super
+      server_options = options.select { |k, _| k.to_sym == :token }
```

Comment:
uhm `server_options = { token: options[:token] }`? Definitely no need to account for a string key in config.